tutorial/westeros/westeros_baseline.ipynb

```diff
@@ -380,9 +380,32 @@
      "slide_type": "fragment"
     }
    },
-   "outputs": [],
-   "source": [
-    "gdp_profile = pd.Series([1., 1.5, 1.9], \n",
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.axes._subplots.AxesSubplot at 0x7fb7cb9c6f28>"
```

Comment:
Please remove notebook output

Reply:
Is there a specific reason we don't include the output?

It's common in [widely-circulated notebooks](https://github.com/jupyter/jupyter/wiki/A-gallery-of-interesting-Jupyter-Notebooks#economics-and-finance)—e.g. [Reinhart & Rogoff replication](http://nbviewer.jupyter.org/github/vincentarelbundock/Reinhart-Rogoff/blob/master/reinhart-rogoff.ipynb)—to include some output. This lets a reader "read" the notebook without having to download it, install software, and re-run the code. Without some output, a notebook can be opaque and fail to clearly explain what the code does; this was my impression on first reading the tutorials.

Reply:
Per conversation in Slack, we will keep notebooks 'bare' in the repo for now, and investigate how to have them automatically rendered as part of the discussion on maintaining multiple versions of documentation.